tests/lean/guessLexFailures.lean.expected.out

```diff
@@ -7,7 +7,27 @@ argument #1 was not used for structural recursion
 
 structural recursion cannot be used
 
-failed to prove termination, use `termination_by` to specify a well-founded relation
-guessLexFailures.lean:12:0-15:31: error: failed to prove termination, use `termination_by` to specify a well-founded relation
-guessLexFailures.lean:22:0-24:31: error: failed to prove termination, use `termination_by` to specify a well-founded relation
-guessLexFailures.lean:30:0-33:31: error: failed to prove termination, use `termination_by` to specify a well-founded relation
+failed to prove termination, possible solutions:
+  - Use `have`-expressions to prove the remaining goals
+  - Use `termination_by` to specify a different well-founded relation
+  - Use `decreasing_by` to specify your own tactic for discharging this kind of goal
+n : Nat
+⊢ Nat.succ n < n
+guessLexFailures.lean:15:14-15:31: error: failed to prove termination, possible solutions:
+  - Use `have`-expressions to prove the remaining goals
+  - Use `termination_by` to specify a different well-founded relation
+  - Use `decreasing_by` to specify your own tactic for discharging this kind of goal
+n : Nat
+⊢ Nat.succ n < n
+guessLexFailures.lean:24:14-24:31: error: failed to prove termination, possible solutions:
+  - Use `have`-expressions to prove the remaining goals
+  - Use `termination_by` to specify a different well-founded relation
+  - Use `decreasing_by` to specify your own tactic for discharging this kind of goal
+n✝ n : Nat
+⊢ n < n✝
+guessLexFailures.lean:33:14-33:31: error: failed to prove termination, possible solutions:
+  - Use `have`-expressions to prove the remaining goals
+  - Use `termination_by` to specify a different well-founded relation
+  - Use `decreasing_by` to specify your own tactic for discharging this kind of goal
+n✝ n : Nat
+⊢ n < Nat.succ n✝
```

Comment:
TODO:

These tests need to be updated/extended to have more than one parameter, else the code that they should test would just be skipped.

Same also for `guessLex.lean`.